mm2src/mm2_test_helpers/dummy_files/iris_nimda_history.json

```diff
@@ -0,0 +1,391 @@
+[
+    {
+        "tx_hex": "0a2a696161317a36303967397a386566346a6c6b63666767333663326c6a67636d6e6135306c32796a726130122a696161316530727838376d646a37397a656a65777563346a6737716c39756432323836673275733866322a0c0a036e696d1205313030303032403838613735653161306166373662636161386266353566313437616266616262313362323964646333316465323161373431363835333966373135643032343240980c",
+        "tx_hash": "B06D02A1449B07E89D15E709C347E0EE72A944687942E5A10467EC096D067FE7",
+        "from": [
+            "iaa1z609g9z8ef4jlkcfgg36c2ljgcmna50l2yjra0"
+        ],
+        "to": [],
+        "total_amount": "0.027385",
+        "spent_by_me": "0.027385",
+        "received_by_me": "0",
+        "my_balance_change": "-0.027385",
+        "block_height": 6175147,
+        "timestamp": 1669934633,
+        "fee_details": null,
+        "coin": "IRIS-NIMDA",
+        "internal_id": "0000000000000000423036443032413134343942303745383944313545373039",
+        "transaction_type": {
+            "CustomTendermintMsg": {
+                "msg_type": "FeeForTokenTx",
+                "token_id": "e13c266c61de5e79cf37a99b94d4d4462d6857fe8a5dbdde9e041ae01b14a7db"
```

Comment:
There are two notes here:
1. `coin` should be `IRIS-TEST`, because the fee is paid in the platform `IRIS-TEST` coin
2. At the next iteration, I'd like to change the transaction type to the following:
* If the tx is a standard `IRIS-NIMDA` token transfer, there should be one `TransactionType::FeeForTokenTx` transaction at the `IRIS-TEST` history, and one `TokenTransfer` transaction at the `IRIS-NIMDA` history
* If the tx is an HTLC `IRIS-NIMDA` transaction, there should be `CustomTendermintMsg: { msg_type: "SendHtlcAmount/ClaimHtlcAmount/SignClaimHtlc" }` same for `IRIS-TEST` and `IRIS-NIMDA` histories.
Or consider using `TransactionType::FeeForTokenTx` for `IRIS-TEST` instead.

So I think we should move [TransactionType::CustomTendermintMsg { FeeForTokenTx }](https://github.com/KomodoPlatform/atomicDEX-API/blob/iris-swap-opt/mm2src/coins/tendermint/mod.rs#L23) to `TransactionType::FeeForTokenTx`.